msk-cfn-sasl-lambda/create-cluster-cfn/MSKSampleStack.yml

```diff
@@ -166,6 +166,7 @@ Resources:
       AutomaticStopTimeMinutes: 30
       Description: "Cloud9 EC2 environment"
       InstanceType: t3.large
+      ImageId: amazonlinux-2023-x86_64
```

Comment:
Thanks for this. It is recommended to have a PR for a single pattern. As this change is for a different pattern, I would request you to create a different PR for this. Look forward to your PR.